### PR TITLE
Enable FIPS testing for SLED - Core maintenance

### PR DESCRIPTION
**Pull request by rfan1**
https://progress.opensuse.org/issues/125195
https://bugzilla.suse.com/show_bug.cgi?id=1208951

- Verification run: 
[Firefox-with fips](https://openqa.suse.de/tests/10633356)
[Firefox-non-fips](http://openqa.suse.de/tests/10633358)
[SLED-others](http://openqa.suse.de/tests/10626751)

### CODE REVIEW

File: schedule/qam/15-SP4/qam-regression-other_fips.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
+name: qam regression other tests with fips enabled for Vendor Affirmation
+schedule:
+- boot/boot_to_desktop
```
Comment (foursixnine) [inline schedule/qam/15-SP4/qam-regression-other_fips.yaml:4]: openjdk is missing

Comment (foursixnine) [reply schedule/qam/15-SP4/qam-regression-other_fips.yaml:4]: Yeah, for both (SLES and SLED)... x11 is not an issue in this case

Comment (rfan1) [reply schedule/qam/15-SP4/qam-regression-other_fips.yaml:4]: Let me do!

Comment (rfan1) [reply schedule/qam/15-SP4/qam-regression-other_fips.yaml:4]: http://openqa.suse.de/tests/10641681#

**Pull request merged**